IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FastenerRailWeld/DocPropertySet.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_FastenerRailWeld" UniqueId="27eae412-12dc-d9f9-29a1-1578ad14fd41" ApplicableType="IfcFastener/WELD" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="BelongingPanel_2I_S0OrPP7Wvu0OPlABgP" />
```

Comment:
Can BelongingPanel be inferred from decomposition structure?

Reply:
Agree, Can be extracted from element assembly it has been aggregated. I have erased the property.6cb18533994026ae118add924a2b65d2f98b8472